.circleci/config.yml

```diff
@@ -90,7 +90,7 @@ jobs:
       - setup_remote_docker: *latest_remote_docker
       - docker_login
       - gradle:
-          args: dockerPushImages
+          args: publish dockerPushImages
```

Comment:
not sure difference between merge-publish and release-publish, but this probably deserves a comment as it is different between them, or fixed to be the same?